docs/en_US/Compressor/Pruner.md

```diff
@@ -265,6 +295,21 @@ Note: ActivationMeanRankFilterPruner is used to prune convolutional layers withi
 
 You can view [example](https://github.com/microsoft/nni/blob/master/examples/model_compress/model_prune_torch.py) for more information.
 
+To enable the dependency-aware mode:
+
+```python
+from nni.compression.torch import ActivationMeanRankFilterPruner
+config_list = [{
+    'sparsity': 0.5,
+    'op_types': ['Conv2d']
+}]
+# dummy_input is necessary for the dependency-aware mode and the
+# dummy_input should be on the same device with the model
+dummy_input = torch.ones(1, 3, 224, 224).cuda()
+pruner = ActivationMeanRankFilterPruner(model, config_list, statistics_batch_num=1, dependency_aware=True, dummy_input=dummy_input)
+pruner.compress()
+```
```

Comment:
there are many one shot pruners supported on NNI, why only some of them support dependency mode? because of implementation issue or other reasons?

Reply:
My bad, the one-shot pruners that inherit from `_StructuredFilterPruner` can support the dependency-aware mode, I will complete the rest as soon as possible.